scripts/setupRegtestEnv.sh

```
@@ -0,0 +1,143 @@
+
+sleep 5
+
+puts "funding boltz lnd"
+address=$($lncli newaddress p2wkh | jq -r .address | cut -c1-45)
+nigiri faucet $address
+
+sleep 5
+
+puts "connecting lnd instances"
+hideOutput=$($lncli connect "$(nigiri lnd getinfo | jq -r .identity_pubkey)"@lnd:9735)
+if [ $($lncli listpeers | jq .peers | jq length) -eq 1 ] && [ $(nigiri lnd listpeers | jq .peers | jq length) -eq 1 ]; then
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)